lang/java/reef-io/src/test/java/org/apache/reef/io/network/util/package-info.java

```diff
@@ -17,6 +17,6 @@
  * under the License.
  */
 /**
- * TODO: Document.
+ * Utility for REEF IO network module.
```

Comment:
This should probably be "Tests for REEF IO network utilities", as this is under test, not main